editor/plugins/audio_stream_editor_plugin.cpp

```diff
@@ -153,10 +159,10 @@ void AudioStreamEditor::_on_input_indicator(Ref<InputEvent> p_event) {
 	Ref<InputEventMouseButton> mb = p_event;
 
 	if (mb.is_valid()) {
```

Comment:
Would it make sense to move the condition for the left mouse button to the whole block?

Like so:
```suggestion
	if (mb.is_valid() && mb->get_button_index() == MOUSE_BUTTON_LEFT) {
```